package.json

```diff
@@ -30,7 +31,7 @@
     "relay:watch": "yarn workspace @animavita/relay watch",
     "start": "yarn workspace @animavita/expo start",
     "storybook": "yarn workspace @animavita/ui storybook",
-    "test": "yarn expo:test && yarn ui:test && yarn graphql:test",
+    "test": "yarn expo:test && yarn ui:test && yarn graphql:test && yarn i18n:test",
```

Comment:
We should refactor this into setting up jest in a monorepo. 

Running `yarn test` in the root should trigger all package's test files under each package's `jest.config.js`